modules/lnd/lnd_wrapper/wrapper.go

```diff
@@ -43,6 +43,8 @@ func LndDeserializeInvoice(cInvoiceStr *C.char) *C.char {
 	sb.WriteString(";AMOUNT=")
 	if invoice.MilliSat != nil {
 		sb.WriteString(fmt.Sprintf("%d", *invoice.MilliSat))
+	} else {
+		sb.WriteString("0")
```

Comment:
Commit adab97de682033f7191585aef64ccc0f2d2e9dea says "Add c-lightning module for differential fuzzing" but it's touching the lnd module as well. In any similar case, it would be great to split the commits and explain it but no need to do this here now, just a tip for future work.